client/extensions/woocommerce/app/shipping/shipping-methods/free-shipping.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Hmmm. Having concrete components for free-shipping, local-pickup, etc is not going to be very DRY - the WooCommerce REST API keeps it pretty generic and we should be able to have a generic shipping method settings component that can be props driven to render the shipping method's settings

See http://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-shipping-methods
and http://woocommerce.github.io/woocommerce-rest-api-docs/#include-a-shipping-method-to-a-shipping-zone and http://woocommerce.github.io/woocommerce-rest-api-docs/#update-a-shipping-method-of-a-shipping-zone